zcash_client_backend/CHANGELOG.md

```diff
@@ -38,6 +39,8 @@ and this library adheres to Rust's notion of
   - `Nullifiers::{orchard, extend_orchard, retain_orchard}`
   - `TaggedOrchardBatch`
   - `TaggedOrchardBatchRunner`
+  - `testing` module
+  - `testing::{'fake_compact_block`, `random_compact_tx`} (moved from `tests` module).
```

Comment:
The `testing` module is new (so the addition of `fake_compact_block` is implicit), the `tests` module is private, and `random_compact_tx` is private. Thus none of this needs to go into the changelog (which is for downstream consumers of the crate's public API):
```suggestion
```
or
```suggestion
  - `testing::fake_compact_block`
```
(Also note the Markdown syntax errors; the backticks should go around the entire name fragment, not the individual function names.)